uportal-war/src/main/java/org/jasig/portal/portlets/portletadmin/PortletDefinitionForm.java

```diff
@@ -555,6 +561,34 @@ public Date getExpirationDateTime() {
 		return cal.getTime();
 	}
 
+
+	/**
+	 * Get the expiration date as a string.  This is
+	 * @return the expiration date as a string
+	 */
+	public String getExpirationDateString() {
+		if (expirationDate == null) {
+			return null;
+		}
+
+		return dateFormat.format(expirationDate);
```

Comment:
Could, but I sometimes get really picky about how much logic to put on each line of code.  I'd generally prefer to write a bit more code and keep it to each line only doing one thing, but that's just personal preference.